### DISCUSSION TITLE
Bring people together

### DISCUSSION THREAD

**Discussion by zapf2000** [bluenviron/mediamtx#3266] [General]
I don't know if you're already there, but there is a discord server where most WebRTC people communicate.
If you like, just join...
https://discord.gg/6TqeBxwb

**Comment by suzp1984** [comment]
> I don't know if you're already there, but there is a discord server where most WebRTC people communicate. If you like, just join... https://discord.gg/6TqeBxwb

Above Link Invite Invalid, maybe expired, that what the `discord` said.

**Reply by amarmashruwala** [reply]
This invite link has expired. Anyone have another link for this discord server? Thanks
